superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -568,12 +570,22 @@ function ChartList(props: ChartListProps) {
         }}
       </ConfirmStatusChange>
 
-      <ImportChartModal
-        show={importingChart}
-        onHide={closeChartImportModal}
+      <ImportModelsModal
+        resourceName="chart"
+        resourceLabel={t('chart')}
+        icon={<StyledIcon name="nav-charts" />}
+        passwordsNeededMessage={t(
+          'The passwords for the databases below are needed in order to ' +
+            'import them together with the charts. Please note that the ' +
+            '"Secure Extra" and "Certificate" sections of ' +
+            'the database configuration are not present in export files, and ' +
+            'should be added manually after the import if they are needed.',
+        )}
         addDangerToast={addDangerToast}
         addSuccessToast={addSuccessToast}
-        onChartImport={handleChartImport}
+        onModelImport={handleChartImport}
+        show={importingChart}
+        onHide={closeChartImportModal}
```

Comment:
did you want to just use `()=> showImportModal(false);` here instead or are you aiming to be more explicit?

Reply:
I've been conditioned by the AirBnB linter to never inline functions in props. My understanding is that it has a performance cost, but that might've changed with functional components?

Reply:
TBH, we used inline functions all the time in the past, so I never gave that concern much weight.. I've heard arguments for/against, but I will research the latest stance on this!

Reply:
Inline functions are fine for functional components. I have yet to see it causing performance issues. The only way it could cause performance issues is when the child component is expensive to re-render and the functional props are the only dependency that is updating between parent re-renders.

If re-render is a real concern, the common practice is to put functions within `useCallback`, or move them out of the functional component. Here is this case there is no additional benefits since the named function is still within the functional component, will be a new variable each time the parent component renders, thus always triggers a re-render of the child component.